wagtailautocomplete/edit_handlers.py

```diff
@@ -21,22 +22,29 @@ def _can_create(page_type):
     from wagtail.wagtailadmin.edit_handlers import BaseFieldPanel
 
     class AutocompletePanel:
-        def __init__(self, field_name, page_type='wagtailcore.Page', is_single=True):
+        def __init__(self, field_name, target_model='wagtailcore.Page', is_single=True, **kwargs):
             # is_single defaults to True in order to have easy drop-in
             # compatibility with wagtailadmin.edit_handlers.PageChooserPanel.
             self.field_name = field_name
-            self.page_type = page_type
+            self.target_model = target_model
             self.is_single = is_single
+            # For compatability with old 'page_type' argument
+            if 'page_type' in kwargs:
+                warnings.warn(
+                    'page_type argument has been replaced with target_model',
+                    DeprecationWarning
+                )
```

Comment:
Absolutely LOVE having these deprecation warnings, but for some reason I'm not seeing them in my console (either on server start or on edit page load). Is that just my environment? Any idea why that might be?

Reply:
yeah, annoyingly by default deprecation warnings are ignored. There's a nice explanation in this so answer: https://stackoverflow.com/a/20960427

Reply:
Huh, but why do I still get Django's deprecation warnings in that case?